src/vrapp/VolumeVisualizationApp.cpp

```diff
@@ -493,25 +493,46 @@ void VolumeVisualizationApp::onGenericEvent(const VRDataIndex& index)
   if (index.getName() == "WindowClose") {
     shutdown();
   }
+
+  if (index.getName() == "WindowSize") {
+    /*
+    some code to update projection matrix
```

Comment:
is this still to do in this PR? or a future PR?

Reply:
I removed it. Buffersize is the one that fixes the frame buffer change event.